Snakemake/config/tsinfer.yaml

```diff
@@ -6,12 +6,6 @@ vcfDir: "/home/jana/Documents/1Projects/HoneybeeDemo/Data/RawVCF/"
 bcftoolsModule: "igmm/apps/bcftools/1.9"
 ancestralAllele: "/home/jana/Documents/1Projects/HoneybeeDemo/Data/AncestralAlleleVcf.txt"
 meta: "/home/jana/Documents/1Projects/HoneybeeDemo/Data/Meta.csv"
-populations:
```

Comment:
I think it is better if we don't pass 'populations' here. It can get really big (I have over 150 pop).

Reply:
Hm, this isn't here anymore on my computer ... Maybe I didn't add the latest version

Reply:
Oh, red means it's deleted

Reply:
right! I didn't see it was a different colour